test/replication-luatest/gh_10727_no_local_rows_test.lua

```diff
@@ -0,0 +1,53 @@
+local t = require('luatest')
+local server = require('luatest.server')
+local proxy = require('luatest.replica_proxy')
+local os = require('os')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.master = server:new{alias = 'master'}
+    cg.master:start()
+    cg.master:exec(function()
+        box.schema.space.create('test', {is_local = true}):create_index('pk')
+        box.space.test:insert{1, 'data'}
+    end)
+end)
+
+g.after_all(function(cg)
+    cg.master:stop()
+end)
+
+local function process_replication_flow(conn, data)
+    local ok, h = pcall(box.iproto.decode_packet, data)
+    if ok and h.group_id == 1 then
+        -- This is the only way, I managed to find to stop the test, when
```

Comment:
Nit: lets avoid pronouns in the comments like "I", "me", "my", etc. The code authorship is going to be not quite easily deductible after it is merged into master.

Reply:
The comment doesn't exist anymore. I got you, for the future